h/models/organization.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_LENGTH = 4
+ORGANIZATION_NAME_MAX_LENGTH = 25
+ORGANIZATION_LOGO_MAX_LENGTH = 3000
+
+
+class Organization(Base, mixins.Timestamps):
+    __tablename__ = 'organization'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    name = sa.Column(sa.UnicodeText(), nullable=False, index=True)
+
+    logo = sa.Column(sa.UnicodeText())
+
+    def __init__(self, **kwargs):
+        super(Organization, self).__init__(**kwargs)
+
+    @property
+    def slug(self):
+        """A version of this organization's name suitable for use in a URL."""
+        return slugify.slugify(self.name)
+
+    @sa.orm.validates('name')
+    def validate_name(self, key, name):
+        if not ORGANIZATION_NAME_MIN_LENGTH <= len(name) <= ORGANIZATION_NAME_MAX_LENGTH:
+            raise ValueError('name must be between {min} and {max} characters '
+                             'long'.format(min=ORGANIZATION_NAME_MIN_LENGTH,
```

Comment:
Super trivial, but the string `long` on the wrapped line is a touch awkward—I imagine you were trying to avoid line lengths > 79...if you're a stickler for that, cool :) but our codebase's linting rules allow for considerably longer line lengths if it's less torturous to keep `long` on line 36.

Reply:
I was but I see your point there it does look a little odd.